### PR TITLE
fix: do not use full tiff files for generating etags

### PR DESCRIPTION
**Pull request by blacha**
as the tiff is often not fully loaded this could cause werid etag generation
instead use just the name of the tiff

### CODE REVIEW

File: packages/lambda-xyz/src/__test__/xyz.test.ts [hunk 1]
```diff
@@ -85,7 +89,7 @@ o.spec('LambdaXyz', () => {
         const res = await handleRequest(request);
         o(res.status).equals(200);
         o(res.header('content-type')).equals('image/webp');
-        o(res.header('eTaG')).equals('w8RW7/QQDHDMiiNDlbACrigBcCEveyedjHn7QMX6kDg=');
+        o(res.header('eTaG')).equals('foo');
```
Comment (jacott) [inline packages/lambda-xyz/src/__test__/xyz.test.ts:92]: You do not appear to be asserting that `generate` is being called with the correct arguments

File: packages/lambda-xyz/src/__test__/xyz.test.ts [hunk 2]
```diff
@@ -18,6 +19,7 @@ o.spec('LambdaXyz', () => {
     const rasterMockBuffer = Buffer.from([1]);
     const origTile256 = Tilers.tile256;
     const origCompose256 = Tilers.compose256;
+    const origTileEtag = TileEtag.generate;
```
Comment (jacott) [inline packages/lambda-xyz/src/__test__/xyz.test.ts:23]: You do not appear to be restoring this

Comment (blacha) [reply packages/lambda-xyz/src/__test__/xyz.test.ts:23]: its restored on line 53

File: packages/lambda-xyz/src/__test__/xyz.test.ts (hunk 1 above)
Comment (blacha) [reply packages/lambda-xyz/src/__test__/xyz.test.ts:92]: I am unsure how much we need to test, considering typescript is already checking that the correct values are passed in. Also the values looked to be completely mocked anyway.

**Pull request merged**